include/nbl/video/IPresentationOracle.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2021 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_VIDEO_I_PRESENTATION_ORACLE__H_INCLUDED__
+#define __NBL_VIDEO_I_PRESENTATION_ORACLE__H_INCLUDED__
+
+#include <nabla.h>
+
+namespace nbl
+{
+	namespace video
+	{
```

Comment:
a) dont `include nabla.h` from any engine header, just include the stuff you need (in this case ILogicalDevice)
b) use `namespace nbl::video` (nested namespace) from now on

Reply:
according to **b)**, is there any difference at all?

Reply:
none, just amount of typing